### PR TITLE
[hedgey-delegate] adds strategy for hedgey vesting contracts

### PR DESCRIPTION
**Pull request by bark4mark**
Changes proposed in this pull request:
Adds hedgey-delegate

This is a simple strategy to call delegatedBalances and apply a multiplier value.

### CODE REVIEW

File: src/strategies/hedgey-delegate/index.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    ])
+  );
+  const result: Record<string, BigNumberish> = await multi.execute();
+
+  return Object.fromEntries(
+    Object.entries(result).map(([address, balance]) => [
+      address,
+      parseFloat(formatUnits(balance, options.decimals)) * options.multiplier
+    ])
+  );
```
Comment (ChaituVR) [inline src/strategies/hedgey-delegate/index.ts:59]: hi @bark4mark I think you can use the `math` strategy with `contract-call` in it :) Can try?

Comment (bark4mark) [reply src/strategies/hedgey-delegate/index.ts:59]: Would it be possible to have this strategy merged? I would like to keep the configuration minimal as I have to create documentation around voting and configuring the smart contracts.

File: src/strategies/hedgey-delegate/index.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'hedgeyfinance';
```
Comment (ChaituVR) [inline src/strategies/hedgey-delegate/index.ts:5]:
```suggestion
export const author = 'hedgey-finance';
```

File: src/strategies/hedgey-delegate/index.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      {
+        internalType: 'uint256',
+        name: 'delegatedBalance',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
```
Comment (ChaituVR) [inline src/strategies/hedgey-delegate/index.ts:33]:
```suggestion
const abi = [
  'function delegatedBalances(address delegate, address token) view returns (uint256 delegatedBalance)'
]
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**